docs/getting-started/Slots.stories.mdx

```diff
@@ -11,4 +12,73 @@ import { paramsBuilder } from "@stories/utils";
 
 # Slots
 
-TBD
+Slots solve a variety of use cases by allowing a component to offer multiple layouts in a declarative way.
+
+It's done by giving a component the ability to offer *named placeholders* which could then be filled by the component consumer to achieve the final layout.
+
+For more information about the *concept of slots*, have a look at web component slots on [MDN](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots#adding_flexibility_with_slots).
+
+In Orbit, slots are defined [implicitly](#) or [explicitly](#) and are fulfilled through the component children.
+
+In the following example, the `<Text>` component implicitly fulfill the `text` slot and the `<LightbulbIcon>` component explicitly fulfill the `end-icon` slot.
+
+```jsx
+<Button>
+    <Text>Cutoff</Text>
+    <LightbulbIcon slot="end-icon" />
+</Button>
+```
+
+To identify which slots are available for a component, view the API section of the component. [Have a look at the button component slots](?path=/docs/button--default-story#button-1).
+
+<Figure url="https://raw.githubusercontent.com/gsoft-inc/sg-orbit/master/docs/getting-started/assets/vertical-rythm-in-action.png" caption="Button api" width="524" height="196" />
+
+## Implicit slots
+
+Most *slotables* Orbit components come with a default slot name.
+
+For example, an `<Icon>` component will implicitly fulfill an `icon` slot and a `<Button>` component will implicitly fulfill a `button` slot.
```

Comment:
Literal translation from French., I would either remove For example or use a more English way of saying this